### PR TITLE
Implementing basic functionality of the simulator and dashboard

### PR DESCRIPTION
**Pull request by muneebafzal**
The mock data generator is created using forms. The data from the form is saved in the database and shown on the dashboard. Note that for now, the data is shown for each of the packet. Later, the data will be shown in the form of graphs when numerous data packages are sent from the simulator.

### CODE REVIEW

File: mysite/settings.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 """
 
 import os
-import django_heroku
+
+# import django_heroku
```
Comment (de846) [inline mysite/settings.py]: Why did you comment this out?

File: mysite/settings.py [hunk 2]
```diff
@@ -115,7 +116,7 @@
 
 STATIC_URL = "/static/"
 
-django_heroku.settings(locals())
+# django_heroku.settings(locals())
```
Comment (de846) [inline mysite/settings.py]: Same - why did you comment this out?

**Comment by de846** [comment]
Your build didn't pass - something because of staticfiles. We'll have to sort that out before this gets merged in.

File: mysite/settings.py [hunk 3]
```diff
@@ -115,7 +116,7 @@
```
Comment (de846) [inline mysite/settings.py]:
```suggestion
STATIC_ROOT = ""
```

File: mysite/settings.py (hunk 1 above)
Comment (muneebafzal) [reply mysite/settings.py]: This was just to work locally. Forgot to uncomment.

File: mysite/settings.py (hunk 2 above)
Comment (muneebafzal) [reply mysite/settings.py]: This is was just to work locally. Forgot to uncomment. Thanks

**Review by muneebafzal — COMMENTED** [review]
LGTM. Thank you

File: mysite/settings.py (hunk 1 above)
Comment (de846) [inline mysite/settings.py]:
```suggestion
import django_heroku
```

File: mysite/settings.py [hunk 4]
```diff
@@ -112,10 +113,11 @@
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/dev/howto/static-files/
-
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
+STATIC_ROOT = os.path.join(PROJECT_DIR, "staticfiles")
 STATIC_URL = "/static/"
 
-django_heroku.settings(locals())
+# django_heroku.settings(locals())
```
Comment (de846) [inline mysite/settings.py]:
```suggestion
django_heroku.settings(locals())
```

**Review by de846 — APPROVED** [review]
LGTM

**Pull request merged**